todo_app/templates/index.html

```diff
@@ -18,9 +18,17 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
+      <ul class="list-group mb-4" style="list-style-type: none;">
         {% for item in items %}
-        <li> {{ item.id}} - {{ item.title }} - {{ item.status }}</li>
+        <li name="" style="width: 400px;margin-bottom: 10px;border: 1px solid #ccc;border-radius: 5px;padding:5px; ">
+          {{ item.title }} 
+          {% if item.status == 'Open' %}
```

Comment:
I'm assuming this matches your list names; we assume `To Do` & `Done` but there's no harm - just watch out for our assumptions in future modules, and make sure you adapt those to your list names!